.github/CODEOWNERS

```diff
@@ -9,10 +9,11 @@
 # explicitly taken by someone else.
 *                               @googleapis/yoshi-go-admins
 
-/bigtable/       @googleapis/api-bigtable @googleapis/yoshi-go-admins
-/datastore/      @googleapis/firestore-dpe @googleapis/yoshi-go-admins
-/errorreporting/ @GoogleCloudPlatform/observability-devx @googleapis/yoshi-go-admins @nicoleczhu
-/firestore/      @googleapis/firestore-dpe @googleapis/yoshi-go-admins
-/logging/        @GoogleCloudPlatform/observability-devx @googleapis/yoshi-go-admins @nicoleczhu
-/storage/        @GoogleCloudPlatform/storage-dpe @googleapis/yoshi-go-admins
-/iot/            @gcseh @GoogleCloudPlatform/api-iot @googleapis/yoshi-go-admins
+/bigtable/**/*.go                     @GoogleCloudPlatform/cloud-native-db-dpes
+/cloudsql/**/*.go                     @GoogleCloudPlatform/infra-db-dpes
+/datastore/**/*.go                    @GoogleCloudPlatform/cloud-native-db-dpes
+/errorreporting/                      @GoogleCloudPlatform/observability-devx @googleapis/yoshi-go-admins @nicoleczhu
+/firestore/                           @GoogleCloudPlatform/cloud-native-db-dpes
+/logging/                             @GoogleCloudPlatform/observability-devx @googleapis/yoshi-go-admins @nicoleczhu
+/storage/**/*.go                      @GoogleCloudPlatform/cloud-storage-dpes
```

Comment:
Should we not keep `@googleapis/yoshi-go-admin`?

Reply:
You can't have cross org teams in the CODEOWNERS (they don't work because the team needs write access to the repo).